docs/UserGuide.adoc

```diff
@@ -396,3 +423,14 @@ e.g. `warning e/20 s/10`
 * *Back to input* : kbd:[Shift] + kbd:[Enter]
 * *To bottom of list* : kbd:[Shift] + kbd:[d]
 * *To top of list* : kbd:[Shift] + kbd:[u]
+
+== Appendix A: File Naming Convention
+
+MediTabs uses the following File Naming Convention when File Name field is used e.g. The `export` command's optional `[FILE_NAME]` field. The File Naming Convention is to ensure consistency and to avoid potential bugs involving file names with different operating systems such as Windows, Linux and MacOS +
+Format: `Start with an Alphabet or Number followed by Alphabets, Numbers, underscore or hyphen`
+
+[IMPORTANT]
+You should follow the File Naming Convention format when specifying the File Name field for any commands that support File Name as a field.
+
+[CAUTION]
+The characters of the File Name cannot contain any spacing or symbols such as `\/:*?"<>|.`
```

Comment:
What about +[]{};!@#$%^&~` etc?

Reply:
I think there are too many symbols to write down as we can also include other ascii or unicode symbols which are not allowed this is why I stated symbols such as .... maybe I will change it to symbols such as but not limited to